codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  notify:
+    after_n_builds: 4
```

Comment:
Took this straight from `pack` but I'm actually not sure why it's 4.

Reply:
Looking at https://docs.codecov.io/docs/notifications#section-preventing-notifications-until-after-n-builds I think actually we want this number to be 2.